pkg/linter/linter.go

```diff
@@ -511,6 +511,84 @@ func (l Linter) Lint(ctx context.Context) (report.Report, error) {
 	return finalReport, nil
 }
 
+// DetermineEnabledRules returns the list of rules that are enabled based on
+// the supplied configuration. This makes use of the linter rule settings
+// to produce a single list of the rules that are to be run on this linter
+// instance.
+func (l Linter) DetermineEnabledRules(ctx context.Context) ([]string, error) {
```

Comment:
The changes in this file are only due to golangcilint now saying public methods should precede private ones